packages/utils/lib/telemetry/tracer.ts

```diff
@@ -1,10 +1,12 @@
-import type { DBEnvironment, DBTeam } from '@nangohq/types';
 import tracer from 'dd-trace';
 
+import type { DBEnvironment, DBTeam } from '@nangohq/types';
+
 export function tagTraceUser({ account, environment }: { account: DBTeam; environment: DBEnvironment }) {
     tracer.setUser({
         id: String(account.id),
         environmentId: String(environment.id),
+        // TODO: use plan
```

Comment:
is it too complicated to add in this PR?

Reply:
The PR is already big, I'll do it just after